### DISCUSSION TITLE
Add translation support

### DISCUSSION THREAD

**Discussion by MagicLike** [JulianPrieber/littlelink-custom#150] [Feature requests]
It would be great to translate LLC into languages other than English.
Are there plans to make this possible / is there already a way?
If not, will it be made possible with Weblate / Crowdin or just with files here on GitHub?

**Comment by JulianPrieber** [comment]
On the list, but will probably never happen.

**Reply by JulianPrieber** [reply]
Accomplishing this is a pretty sizable effort. I'd have to move every bit of text in the entire application to a separate text file. 

This is technically fairly easy to do, but would take a quite a while.

**Reply by MagicLike** [reply]
I would volenteer to do that from time to time...
